src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/commands.py

```diff
@@ -130,7 +130,11 @@ def transform_sku_for_table_output(skus):
 cli_command(__name__, 'vm resize', custom_path.format('resize_vm'), no_wait_param='no_wait')
 cli_command(__name__, 'vm capture', custom_path.format('capture_vm'))
 cli_command(__name__, 'vm open-port', custom_path.format('vm_open_port'))
-cli_command(__name__, 'vm format-secret', custom_path.format('get_vm_format_secret'))
+cli_command(__name__, 'vm format-secret', custom_path.format('get_vm_format_secret'), deprecate_info='az vm secret format')
+cli_command(__name__, 'vm secret format', custom_path.format('get_vm_format_secret'))
+cli_command(__name__, 'vm secret add', custom_path.format('add_vm_secret'))
+cli_command(__name__, 'vm secret show', custom_path.format('show_vm_secret'))
+cli_command(__name__, 'vm secret delete', custom_path.format('remove_vm_secret'))
```

Comment:
Typically in the CLI, create/delete are paired for resources and add/remove are paired for collections.  I would either change the add to create or delete to remove.